joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller.h

```diff
@@ -246,8 +246,41 @@ class JointTrajectoryController : public controller_interface::Controller<Hardwa
    */
   void setHoldPosition(const ros::Time& time, RealtimeGoalHandlePtr gh=RealtimeGoalHandlePtr());
 
+  /**
+   * \brief Internal update function.
+   *
```

Comment:
Could we please remove these newlines in the brief doc? and also between `\brief` and `\note` blocks

Reply:
Done